### ISSUE TITLE
Reported as malicious software distribution

### ISSUE THREAD

**Issue by drakernoise** [PrestaShop/erpillicopresta#4]
Hi, 
My host provider ( Strato ) has reported the file( modules/erpillicopresta/config/control.php ) to be hacked and used to distribute malware.

Would it be possible?

I've compared my control.php file to the original uploaded by you and they match, no changes were made...

I'm reporing to you, just in case

Best Regards

**Comment by PierreRambaud** [comment]
This is just because the file is obfuscate. Don't worry, everything is ok.

**Comment by drakernoise** [comment]
Yeah I guessed, the fact they blocked my domains is upseting though... will see how long they spend on realizing the actual facts so they open my domains again.

Hope it can help you somehow

**Comment by ttoine** [comment]
Thanks for the report. This kind of obfuscated file is sometime used by module developers for the activation of the user license.

However, if this module is open source (ex: distributed with an AFL license like PrestaShop's modules), you should have the entire source code of the module, and there should be no obfuscated code. That's maybe why your host provider has reported this file.
